### PR TITLE
REVIEW: Feature/rename gem

### PR DESCRIPTION
**Pull request by smdern**

### CODE REVIEW

File: lib/wemux-pair.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+require_relative 'wemux_pair'
```
Comment (aaronjensen) [inline lib/wemux-pair.rb]: this should be `wemux/pair`

File: lib/wemux/pair/commands/start_client_session.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@
 require_relative '../client_tunnel'
 require_relative '../pair_connection'
 
-module WePair
+module Wemux::Pair
```
Comment (aaronjensen) [inline lib/wemux/pair/commands/start_client_session.rb]: :rotating_light: these should all take the form of

```
module Wemux
  module Pair
  end
end
```

File: spec/wemux/pair/commands/command_spec.rb [hunk 3]
```diff
@@ -1,29 +1,29 @@
 require 'spec_helper'
 
-require 'wepair/commands/command'
+require 'wemux/pair/commands/command'
 
-describe WePair::Commands::Command do
+describe Wemux::Pair::Commands::Command do
```
Comment (aaronjensen) [inline spec/wemux/pair/commands/command_spec.rb]: :yellow_heart: if you put specs inside `module Wemux::Pair` you don't need to namespace all the things.

File: lib/wemux/pair/pair.rb [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+require_relative 'commands/start_client_session'
+require_relative 'commands/check_for_wemux'
+require_relative 'commands/check_for_pair_user'
+require_relative 'commands/create_sample_config'
+require_relative 'commands/set_pow_host'
+require_relative '../pair'
+
+module Wemux
+  module Pair
+    class Pair
```
Comment (smdern) [inline lib/wemux/pair/pair.rb]: @aaronjensen should this still be Pair::Pair? or should it be moved into `lib/wemux/pair.rb` ?

Comment (smdern) [inline lib/wemux/pair/pair.rb]: or maybe rename it to `run.rb` or `command.rb`.. ?

Comment (aaronjensen) [inline lib/wemux/pair/pair.rb]: probably runner

File: README.md [hunk 5]
```diff
@@ -1,12 +1,17 @@
-# WePair
+# Wemux::Pair
 
-TODO: Write a gem description
+Create a tunnel to a host machine using an intermediate, commonly accessible
```
Comment (aaronjensen) [inline README.md]: this should be a simpler explanation. Something along the lines of "Firewall punching remote pairing with wemux made easy" or whatever

File: README.md [hunk 6]
```diff
@@ -1,12 +1,17 @@
...
+box. Have other people join your wemux session.
+
+Forwards rails server port and adds to client's pow (if configured and
+available).
 
 ## Installation
```
Comment (aaronjensen) [inline README.md]: I'd add a requirements section above this... since having a box is a pretty big requirement :)

File: bin/pair [hunk 7]
```diff
@@ -1,14 +1,5 @@
 #!/usr/bin/env ruby
 
-=begin
-First time set up for hosts (assuming OSX):
+require_relative '../lib/wemux-pair'
```
Comment (aaronjensen) [inline bin/pair]: :yellow_heart: require `wemux/pair` instead, wemux-pair is really only there for bundler

**Comment by smdern** [comment]
@aaronjensen anymore feedback?

**Comment by aaronjensen** [comment]
:shipit:

**Pull request merged**